eng/pipelines/runtime-extra-platforms-non-wasm.yml

```diff
@@ -0,0 +1,364 @@
+jobs:
```

Comment:
I expect that people will want to create more subsets from this, for similar reasons you are creating the wasm subset.

Would it make sense to call this file `runtime-other` so that we do not end up with odd names as more subsets are created?